utils/protobuf-schema-utilities/src/test/proto/decimal/decimal.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package decimal;
+
+option java_package = "decimal";
+option java_outer_classname = "ProtobufSchemaDecimal";
+
+// Protobuf standard library / WellKnownTypes don't provide a Decimal type.
+// This posses a challenge to represent Decimals that cannot be accurately represented
+// using float or double types. This Proto provides a way to represent Decimals for
+// customers using Glue Schema Registry.
+// Related issue: https://github.com/protocolbuffers/protobuf/issues/4406
+message Decimal {
```

Comment:
Do we need to duplicate this? Can the one in sources be used? Also, please remove references to Glue Schema Registry.

Reply:
I'm not sure - Kexin has both in her metadata changes? I will try with just one